src/windows/webview.c

```diff
@@ -684,11 +756,16 @@ bool mty_webview_is_steam(void)
 
 bool mty_webview_is_available(void)
 {
-	HMODULE webview = webview_load_dll();
-	if (webview) {
-		FreeLibrary(webview);
-		return true;
-	}
+	wchar_t path[MTY_PATH_MAX] = {0};
 
-	return false;
+	bool have_path = webview_dll_path(path, false);
+	if (!have_path)
+		have_path = webview_dll_path(path, true);
+
+	if (!have_path)
+		return false;
+
+	// Loading the lib would be ideal to be sure, but repeated loads eventually cause issues from Windows not un-reserving memory.
+	// https://forums.codeguru.com/showthread.php?60548-Is-there-a-limit-on-how-many-times-one-can-load-(and-free)-the-same-DLL-in-a-process&p=156821#post156821
+	return PathFileExists(path);
```

Comment:
With the DLL's checking if the path exists, adjusted path:

```suggestion
	// Loading the lib would be ideal to be sure, but repeated loads eventually cause issues from Windows not un-reserving memory.
	// https://forums.codeguru.com/showthread.php?60548-Is-there-a-limit-on-how-many-times-one-can-load-(and-free)-the-same-DLL-in-a-process&p=156821#post156821
	return false;
```